### PR TITLE
Update 500.html

### PR DESCRIPTION
**Pull request by dpakprajul**
change the contact information: email address and phone number

### CODE REVIEW

**Comment by Andreasr28** [comment]
Changed to which address and number?

File: templates/500.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1>{% trans 'Maintenance ongoing' %}</h1>
             {% trans 'we are currently working on the Geoportal.' %} <br>
             {% trans 'The service will be back as fast as possible.' %}
             <hr>
-            {% trans 'Helpdesk contact:' %} +49 (0) 261/492-300 | helpdesk@vermkv.rlp.de
+            {% trans 'Helpdesk contact:' %} +49 (0) 611/535-5513 | verHVBG-Geoportal-Root@hvbg.hessen.de
```
Comment (dpakprajul) [inline templates/500.html:18]: @Andreasr28 changed to this.

Comment (Andreasr28) [reply templates/500.html:18]: Ok .. than this email pls : gdi-hessen@hvbg.hessen.de

Comment (Andreasr28) [reply templates/500.html:18]: Not worth all the individual e-mail effort  .. Leave the root address in it

**Review by karlbrink — APPROVED** [review]

**Pull request merged**